.github/workflows/CI.yml

```
@@ -0,0 +1,62 @@
+name: CI
+
+# Trigger the workflow on push or pull request
+on:
+  - push
+  - pull_request
+
+jobs:
+  # The job running the package tests.
+  test:
+    name: ${{ matrix.gap-branch }} ${{ matrix.ABI }} - HPCGAP ${{ matrix.HPCGAP }} - ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        gap-branch:
+          - master
+          - stable-4.11
+          - stable-4.10
+        GAP_PKGS_TO_BUILD: ['profiling']
+        HPCGAP: ['no']
+        ABI: ['']
+        include:
+          - os: ubuntu-latest
+            gap-branch: master
+            GAP_PKGS_TO_BUILD: ''
+            HPCGAP: 'yes'
+            NO_COVERAGE: 'yes'
+          - os: ubuntu-latest
+            gap-branch: master
+            ABI: 32
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: gap-actions/setup-gap-for-packages@v1
+        with:
+          GAPBRANCH: ${{ matrix.gap-branch }}
+          GAP_PKGS_TO_BUILD: ${{ matrix.GAP_PKGS_TO_BUILD }}
+          HPCGAP: ${{ matrix.HPCGAP }}
+          ABI: ${{ matrix.ABI }}
```

Alphabetic order makes longterm maintenance easier (but more importantly, it should allow me to re-trigger a build of this PR ;-) )
```suggestion
          ABI: ${{ matrix.ABI }}
          GAPBRANCH: ${{ matrix.gap-branch }}
          GAP_PKGS_TO_BUILD: ${{ matrix.GAP_PKGS_TO_BUILD }}
          HPCGAP: ${{ matrix.HPCGAP }}
```